web/blog/the-new-rest-api/index.md

```diff
@@ -157,16 +170,191 @@ vast web server --mode=mtls
 ## Usage Examples
 
 Now that you know how we put the REST API together, let's look at some
-end-to-end examples. We built the REST API for two reasons::
+end-to-end examples.
+
+### See what's inside VAST
+
+One straightforward example is checking the number of records in VAST:
+
+```bash
+curl "https://vast.example.org:42001/api/v0/status?verbosity=detailed" \
+  | jq .index.statistics
+```
+
+```json
+{
+  "events": {
+    "total": 8462
+  },
+  "layouts": {
+    "zeek.conn": {
+      "count": 8462,
+      "percentage": 100
+    }
+  }
+}
+```
+
+### Perform a HTTP health check
+
+The `/status` endpoint can also be used as a HTTP health check in
+`docker-compose`:
+
+```yaml
+version: '3.4'
+services:
+  web:
+    image: tenzir/vast
+    environment:
+      - "VAST_START__COMMANDS=['web server --mode=dev']"
+    ports:
+      - "42001:42001"
+    healthcheck:
+      test: curl --fail http://localhost:42001/status || exit 1
+      interval: 60s
+      retries: 5
+      start_period: 20s
+      timeout: 10s
+```
+
+### Run a query
+
+The other initial endpoints can be used to get data out of VAST. For example, to
+get up to two `zeek.conn` events which connect to the subnet `192.168.0.0/16`, using
+the VAST query expression `net.src.ip in 192.168.0.0/16`:
 
-1. Enable third parties to integrate with VAST
-2. Develop our own web frontend
+```bash
+curl "http://127.0.0.1:42001/api/v0/export?limit=2&expression=net.src.ip%20in%20192.168.0.0%2f16"
+```
+
+```json
+{
+  "version": "v2.4.0-457-gb35c25d88a",
+  "num_events": 2,
```

Comment:
Should this key be using a dash rather than an underscore in the future? We use dashes in the status output keys, for example.

Reply:
Created an issue for that.